BUILDING.md

```diff
@@ -566,9 +566,9 @@ to run it again before invoking `make -j4`.
 
 * [Python 3.10](https://www.microsoft.com/en-us/p/python-310/9pjpw5ldxlz5)
 * The "Desktop development with C++" workload from
-  [Visual Studio 2019](https://visualstudio.microsoft.com/downloads/) or
+  [Visual Studio 2019](https://visualstudio.microsoft.com/vs/older-downloads/#visual-studio-2019-and-other-products) or
   the "C++ build tools" workload from the
-  [Build Tools](https://visualstudio.microsoft.com/downloads/#build-tools-for-visual-studio-2019),
+  [Build Tools](https://aka.ms/vs/16/release/vs_buildtools.exe),
```

Comment:
I cannot find a visual studio 2019 c++ build tools link from https://visualstudio.microsoft.com/downloads.

Using https://aka.ms/vs/16/release/vs_buildtools.exe link to download visual studio 2019 build tools is documented at 
https://docs.microsoft.com/en-us/visualstudio/install/build-tools-container?view=vs-2019#create-and-build-the-dockerfile

<img width="836" alt="image" src="https://user-images.githubusercontent.com/12343178/170863362-83628014-9d68-4a8d-b9f6-625c7adfb32d.png">